src/threesixty.js

```diff
@@ -154,8 +190,13 @@ var ThreeSixty = (function (window) {
         };
 
         self.update = function () {
-            container.style.backgroundPositionX = -(index % options.perRow) * options.width + 'px';
-            container.style.backgroundPositionY = -Math.floor(index / options.perRow) * options.height + 'px';
+            if (options.array) {
+                $(container).find('img').attr("src", arrayImages[index].src);
```

Comment:
We should avoid using jQuery here. Something like this can be used instead: https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute